### DISCUSSION TITLE
Bug: Cannot redefine property - but it's only included once in the source...

### DISCUSSION THREAD

**Discussion by Inlustra** [vercel/next.js#14796] [Help]
Hey guys,

As the title says, I'm seeing this bug:
```
TypeError: Cannot redefine property: REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED
    at Function.defineProperty (<anonymous>)
    at eval (webpack-internal:///./node_modules/@react-navigation/core/lib/commonjs/BaseNavigationContainer.js:181:8)
```

I'm not 100% sure that this bug is caused by NextJS, but I wanted to discuss and see if maybe anyone had some insight.
This redefined property is only included once in the file and is happening during SSR (Not sure if it's also happening during client side, any easy way for me to check?)

**Comment by Inlustra** [comment]
Some extra context, this is happening both during build and development.
I've also check yarn.lock and can see that I've not pulled in extra versions of this package 🤔

**Comment by jamesmosier** [comment]
Are you able to share some example code?

**Comment by Inlustra** [comment]
Not exactly, I can't even really work out what's causing it in this massive project, figured it might be good to ask around see if anyone has any ideas...

Perhaps this is expected and fine? 
IE: The library needs to try/catch the offending code:

```
/**
 * Migration instructions for removal of devtools from core
 */
Object.defineProperty(
  global,
  'REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED',
  {
    set(_) {
      console.warn(
        "Redux devtools extension integration can be enabled with the '@react-navigation/devtools' package. For more details, see https://reactnavigation.org/docs/devtools"
      );
    },
  }
);
```
